epochStart/metachain/economics.go

```diff
@@ -130,8 +135,18 @@ func (e *economics) ComputeEndOfEpochEconomics(
 	maxBlocksInEpoch := core.MaxUint64(1, roundsPassedInEpoch*uint64(e.shardCoordinator.NumberOfShards()+1))
 	totalNumBlocksInEpoch := e.computeNumOfTotalCreatedBlocks(noncesPerShardPrevEpoch, noncesPerShardCurrEpoch)
 
+	supplyToUseForRewardsperBlock := e.genesisTotalSupply
+	if metaBlock.Epoch >= e.accRewardsEnableEpoch {
+		supplyToUseForRewardsperBlock = prevEpochEconomics.TotalSupply
```

Comment:
this will still compound daily so it will end up larger than initially configured. 
To have the exact inflation configured, you need to either use the same supply for the entire year when computing rewards, or dynamically decrease the starting inflation rate daily to have a constant emission each epoch.